exercises/concept/plane-tickets/.docs/introduction.md

```diff
@@ -0,0 +1,96 @@
+# Instructions
+
+A generator in Python is a _callable function_ that returns a [lazy iterator](https://en.wikipedia.org/wiki/Lazy_evaluation).
+
+_Lazy iterators_ are similar to `lists`, and other `iterators`, but with one key difference: They do not store their `values` in memory, but _generate_ their values when needed.
+
+## Constructing a generator
+
+Constructing a `generator` is a bit different than a normal `function`. You will need the `yield` expression, which we will go into depth with [later](#the-yield-expression).
```

Comment:
```suggestion
Generators are constructed much like other  functions, but require a [`yield` expression](#the-yield-expression), which we will explore  in depth a bit later.
```

Reply:
```suggestion
Generators are constructed much like other functions, but require a [`yield` expression](#the-yield-expression), which we will explore in depth a bit later.
```

Reply:
The suggestion above just removes some "double space" between words.